modules/unsupported/geojson-core/src/main/java/org/geotools/data/geojson/GeoJSONWriter.java

```diff
@@ -80,7 +80,7 @@ public class GeoJSONWriter implements AutoCloseable {
             FastDateFormat.getInstance(DEFAULT_DATE_FORMAT, DEFAULT_TIME_ZONE);
 
     /** Maximum number of decimal places (see https://xkcd.com/2170/ before changing it) */
-    private int maxDecimals = 4;
+    private int maxDecimals = 6;
```

Comment:
While we're at it, can we use JtsModule.DEFAULT_MAX_DECIMAL here?

Reply:
fixed